docs/get-started/install.md

```diff
@@ -2,8 +2,65 @@
 title: 'Install'
 ---
 
-Test content
+Use the Storybook CLI to install it in a single command. Run this inside your existing project’s root directory:
 
-![optimize](./optimize.gif)
+```sh
+npx sb init
+```
+<div style="background-color:#F8FAFC">
+TODO: vet this per feedback (https://github.com/storybookjs/storybook/pull/11632#discussion_r458498081)
 
-## Heading
+</div>
+
+Storybook needs to be installed into a project that is already configured. For instance with:
+
+- 📦 [Create React App](https://reactjs.org/docs/create-a-new-react-app.html)
+- 📦 [Vue Cli](https://cli.vuejs.org/)
+
+
+Or any other tooling available. 
```

Comment:
Put this inside the expandable section reference above. Add it to the list instead of having it as a paragraph.

Reply:
Fixed with latest commit let me know if you're ok with it